src/jdk.jfr/share/classes/jdk/jfr/internal/instrument/JDKEvents.java

```diff
@@ -300,4 +308,14 @@ private static void emitDirectBufferStatistics() {
         DirectBufferStatisticsEvent e = new DirectBufferStatisticsEvent();
         e.commit();
     }
+
+    private static void emitInitialSecurityProperties() {
```

Comment:
Is the `Security` class loaded and have the properties always been populated at this point? ProtectionDomain` doesn't reference the `Security` class AFAICT.